armbian/base/config/wifi/wlan0.conf

```diff
@@ -0,0 +1,4 @@
+auto wlan0
+iface wlan0 inet dhcp
+  wpa-ssid XXXXXX
```

Comment:
(Optional) We could also use something like `{wpa-password}` as a placeholder value, to more clearly mark that this is not a credential but a place where a script should insert a credential.

C.f how Shesek updates placeholders in `spark-wallet.service`:
* https://github.com/shesek/spark-wallet/blob/master/doc/startup-systemd.md#adding-spark-to-startup-with-systemd

(And elsewhere.)

Reply:
Interesting, very focused `sed`, good to know. I still prefer to search for a keyword and replace the whole line. I can use the placeholders anyway, to make their nature more clear.